pyproject.toml

```
@@ -0,0 +1,63 @@
+[tool.poetry]
+name = "companion-docker"
+version = "0.0.1"
+description = "Blue Robotics companion computer project"
```

```suggestion
description = "Blue Robotics Companion computer project"
```
hah. tjhe code suggestion feature is back